### DISCUSSION TITLE
Switching over from AC to g-helper

### DISCUSSION THREAD

**Discussion by BladeofMillenia** [seerge/g-helper#719] [Q&A] [answered]
Hello, 
I am looking to switch from Armory Crate to g-helper,  the main reason for this is whenever I download armoury crate my keyboard backlighting just doesn't work anymore. As a result I have had to reset my laptop and not download armoury crate and only using the applications that allow me to access and use the hot keys related to the applications that come with armoury crate such as Aura and fan control. This was working for about half a year until those stopped working just a week ago. Therefore I have redownloaded armoury crate to regain access to fan control. This has disabled my keyboard backlighting and yes I have tried every method to get it back but it the root cause is armoury crate. I am going to try using g-helper to see if it resolves any of the issues. I have a few questions though which I haven't been able to find on FAQ. 
- Can you bind opening g-helper to the key that is being used to open armoury crate? 
- Do the armoury crate related fn-keys work when solely using g-helper? 
- Should I download g-helper and delete armoury crate or do you recommend that I reset my laptop and just download g-helper?
That is all.

**Comment by Wolper321** [comment, marked as answer]
1)+2) Yes, and you can reconfigure them.
![image](https://github.com/seerge/g-helper/assets/12784148/a2eb29ee-5c06-4aa3-a53c-200c493f8eb6)
3) You can just download G-helper and look if it suits you. If you think it's ok you can delete AC(Uninstall tool) and MyASUS(casual uninstall). Afterwards, remember to go to Extra--->Stop Asus services.

You can go back painlessly if you don't like it.

**Reply by BladeofMillenia** [reply]
THANK YOU VERY MUCH, THE KEYBOARD LIGHTING WORKS NOW, THANK YOU WOLPER AND THE CREATOR OF THIS GENIUS APPLICATION.

**Reply by BladeofMillenia** [reply]
Hold up, I am having trouble binding g-helper to the rog key, can someone explain the process? I would also like the armoury crate installation to stop popping up as well. Sorry I am really not good with coding and configuring.

**Comment by seerge** [comment]
@BladeofMillenia hello, everything is possible. But all your questions are already answered in FAQ on main page
https://github.com/seerge/g-helper#question-faq

**Reply by BladeofMillenia** [reply]
What exactly are window services and where can I access it?

**Reply by seerge** [reply]
You can stop all asus services from g-helpers Extra button -> Stop (and start them back as well)

**Reply by BladeofMillenia** [reply]
Ok THANK YOU VERY MUCH
